### ISSUE TITLE
Incompatible with wavey capes.

### ISSUE THREAD

**Issue by CuteistFox** [redstone-development/Hysentials-java#4]
This is a problem with both mods having a `CapeHolder.simulate`. The best way to solve this is to remove the wavey capes from this mod and instruct users to get wavey capes from modrinth

**Comment by sinender** [comment]
Yeah there have been too many complaints about wavey capes as a default feature so I will probably just remove it entirely. Will need to make sure I am able to still use custom cape rendering with wavey capes.

**Comment by sinender** [comment]
Wavey capes are now removed [commit](https://github.com/redstone-development/Hysentials-java/commit/55e97e11026b1d01e34ee3efaded485b93e955af).

**Comment by sinender** [comment]
Seems to be a bug with essentials, will need to look into more.
